tests/system/feature_store/test_feature_store.py

```diff
@@ -4202,6 +4202,36 @@ def test_get_online_features_after_ingest_without_inference(self):
                 f"store://feature-vectors/{self.project_name}/my-vector:latest"
             )
 
+    def test_ingest_with_rename_columns(self):
+        csv_path = str(self.assets_path / "fields_with_space.csv")
+        run_id = str(uuid.uuid4())
+        name = f"test_ingest_with_rename_columns_{run_id}"
+        data = pd.read_csv(csv_path)
+        expected_result = data.copy().rename(
+            columns={"city of birth": "city_of_birth"}
+        )[["city_of_birth"]]
+        feature_set = fstore.FeatureSet(
+            name=name,
+            entities=[fstore.Entity("name")],
+        )
+        fstore.preview(
+            feature_set,
+            data,
+        )
+        result = fstore.ingest(
+            feature_set, data, run_config=fstore.RunConfig(local=True)
+        )
+        feature_vector = fstore.FeatureVector(
+            name=run_id, features=[f"{self.project_name}/{name}.*"]
+        )
+        training_data = fstore.get_offline_features(feature_vector).to_dataframe()
```

Comment:
Why is this called `training_data`? Doesn't seem like this test has anything to do with model training.